src/auth/googleauth.ts

```diff
@@ -812,13 +812,16 @@ export class GoogleAuth {
     const id = `projects/${projectId}/serviceAccounts/${creds.client_email}`;
     const res = await this.request<SignBlobResponse>({
       method: 'POST',
-      url: `https://iam.googleapis.com/v1/${id}:signBlob`,
-      data: {bytesToSign: crypto.encodeBase64StringUtf8(data)},
+      url: `https://iamcredentials.googleapis.com/v1/{name=${id}}`,
```

Comment:
If I understand correctly, we're moving from the `iam` API to sign, rather than the `iamcredentials` API.

Are we sure these API surfaces are identical, or should be consider making this a breaking change? CC: @frankyn

Reply:
iamcredentials is the v2 of the iam API. I synced with the IAM team on which API libraries should use. iamcredentials works in the same way for this specific scenario and isn't impacted by the requests having to be routed back to US region (which is the case with the existing iam API).